doc/source/serve/tutorials/serve-ml-models.md

```diff
@@ -77,7 +77,7 @@ TypeError: Descriptors cannot not be created directly.
      2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).
 ```
 
-You can downgrade the protobuf package to 3.20.x or lower in your Docker image, or tell Ray to do it at runtime by specifying a [runtime environment](runtime-environments):
+In this case, downgrade the protobuf package to 3.20.x or lower in your Docker image, or tell Ray to do it at runtime by specifying a [runtime environment](runtime-environments):
```

Comment:
@architkulkarni Same question about the protobuf issue here.

Reply:
When I hear back from Archit, I'll open a separate PR if needed.